packages/capsule/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/capsule",
-  "version": "2.0.1",
+  "version": "2.0.1-alpha.1",
```

Comment:
```suggestion
  "version": "2.0.2-alpha.1",
```

Reply:
Please also bump this version in the packages/demo and confirm everything works from the internal demo. Will want to update this version in the docs site too docs/package.json